src/ui/index.js

```diff
@@ -101,7 +101,10 @@ export default async (kbnServer, server, config) => {
           includeUserProvidedConfig
         }),
         bundlePath: `${config.get('server.basePath')}/bundles`,
+        // #{i18n('KEY')} -> value ("Hi!")
         i18n: key => _.get(translations, key, ''),
+        // var v = #{i18n_var('KEY')} -> var v = "value (\"Hi!\")"
+        i18n_singleQuoteVar: key => _.get(translations, key, '').replace(/'/g,'\\\''),
       });
```

Comment:
Ahhhh ..  meant functions!  Just trying to avoid having to make changes if another means possible.